SourcerySwift/Sources/SwiftTemplate.swift

```diff
@@ -240,7 +240,7 @@ open class SwiftTemplate {
                                                        arguments: arguments,
                                                        currentDirectoryPath: buildDir)
 
-        if compilationResult.exitCode != 0 || !compilationResult.error.isEmpty {
+        if compilationResult.exitCode != EXIT_SUCCESS {
```

Comment:
I raised #1040 separately to track this. If it needs a bigger discussion, I can revert this here and move it out to a different PR, but right now I need it to get the tests to pass 😄